src/main/java/org/jitsi/videobridge/PotentialPacketHandler.java

```diff
@@ -30,9 +30,16 @@
     boolean wants(PacketInfo packet, String sourceEpId);
 
     /**
-     * Send the given 'packet' (which came from 'source')
+     * Send the given rtp 'packet' (which came from 'source')
```

Comment:
I wonder  if we need separate methods for RTP and RTCP here, since they both take a PacketInfo. Presumably we'd also call `wants` for RTCP to avoid unnecessary copying, so one method for both makes sense to me

Reply:
shall I rename sendRtp to sendRtpRtcp (or something else) and remove the sendRtcp method?

Reply:
I think we should keep the separate methods. The way we forward them is pretty different and I think it's clearer to have the two handlers distinct from one another, in the new code we treat rtcp and rtp differently.

Reply:
not sure what to implement, i'm personally fine with either @bgrozev @bbaldino can you please guys reach some consensus?

Reply:
I guess I don't see the commonality between the 2 in terms of implementation. I don't see "wants" even as applying to rtcp...rtcp isn't about wanting, it's about routing it to the single place it must go, not asking everywhere if they want it like rtp

Reply:
Because of this, for example, that clone logic doesn't even apply to rtcp so why do we care about reusing it?

Reply:
"it's about routing it to the single place it must go"

No, SRs are being forwarded to everyone and the BitrateController "wants" only the SRs for the SSRC that it is currently forwarding (out of the simulcast set)

Reply:
Ok, but it's not true for pli/fir. It just seems like we're going to have a method that has to handle rtp and rtcp differently anyway, so why not just have the two methods/paths since they do differ.

Reply:
Which of the 3 points above do you disagree with?

I agree that we have to handle them differently *somewhere*, but I think we should do it after we re-use the common logic, and without complicating the wants/send interface

Reply:
I think 1 and 3. I don't think reusing the clone logic will apply directly, pli/fir will only go to 1 place and we don't want to clone and will route totally differently. I assume we need to rewrite the srs so we do need to clone there? Or will we just rebuild a new one?

Reply:
I think 1 and 3. I don't think reusing the clone logic will apply directly, pli/fir will only go to 1 place and we don't want to clone and will route totally differently. I assume we need to rewrite the srs so we do need to clone there? Or will we just rebuild a new one?

Reply:
If we did combine, we'd have a when that basically treated rtp and rtcp srs one way (assuming srs are the same?), Pli/fir another way...is that right?

Reply:
1. By inconsisntent I mean the following: the `wants` path takes a `PacketInfo` and does the RTP/RTCP check internally. The `sendPath` also takes a `PacketInfo`, but the RTP/RTCP check happens prior. 
3. Yes, we `clone()` RTCP SRs. And while not critical it would be nice to avoid the last one (we already have code to do it!).

To clarify, here is what I'm suggesting (updated to handle the FIR/PLI case optimally)
```
interface PotentialPacketHandler{
boolean wants(PacketInfo)
void send(PacketInfo)
}

Conference.sendOut(PacketInfo, Source) {
  <<<existing logic for fanning out without an extra clone(), calling the single send() of PPH>>>
}

Conference.handleIncomingRtp(PacketInfo, Source){
  sendOut(PacketInfo, Source)
}

Conference.handleIncomingRtcp(PacketInfo, Source){
  if (PacketInfo is RTCP FB) {
     endpoint = find(PacketInfo)
     if(endpoint.wants(PacketInfo, Source) endpoint.send(PacketInfo, Source)
  }
  else {
    sendOut(PacketInfo, Source)
  }
}
```

Reply:
I'm good with that

Reply:
i've updated the code